.ci/scripts/calculate_jobs.py

```diff
@@ -58,17 +62,67 @@
         }
     )
 
-no_extra_tests = [
+trial_no_extra_tests = [
     {
         "python-version": "3.7",
         "database": "sqlite",
         "extras": "",
     }
 ]
 
-print("::group::Calculated jobs")
-print(json.dumps(sqlite_tests + postgres_tests + no_extra_tests, indent=4))
+print("::group::Calculated trial jobs")
+print(
+    json.dumps(
+        trial_sqlite_tests + trial_postgres_tests + trial_no_extra_tests, indent=4
+    )
+)
+print("::endgroup::")
+
+test_matrix = json.dumps(
+    trial_sqlite_tests + trial_postgres_tests + trial_no_extra_tests
+)
+print(f"::set-output name=trial_test_matrix::{test_matrix}")
+
+
+# First calculate the various sytest jobs.
+#
+# For each type of test we only run on focal on PRs
+
+
+sytest_tests = [
+    {
+        "sytest-tag": "focal",
+    },
+    {
+        "sytest-tag": "focal",
+        "postgres": "postgres",
+    },
+    {
+        "sytest-tag": "focal",
+        "postgres": "mulit-postgres",
+        "workers": "workers",
+    },
+]
+
+if not IS_PR:
+    sytest_tests.extend(
+        [
+            {
+                "sytest-tag": "testing",
+                "postgres": "postgres",
+            },
+            {
+                "sytest-tag": "buster",
+                "postgres": "mulit-postgres",
+                "workers": "workers",
+            },
+        ]
+    )
+
+
+print("::group::Calculated sytest jobs")
+print(json.dumps(sytest_tests, indent=4))
 print("::endgroup::")
 
-test_matrix = json.dumps(sqlite_tests + postgres_tests + no_extra_tests)
-print(f"::set-output name=test_matrix::{test_matrix}")
+test_matrix = json.dumps(sytest_tests)
+print(f"::set-output name=sytest_test_matrix::{test_matrix}")
```

Comment:
I think you still want to output the trial test matrix?

Reply:
That is still happening above